tests/settings.py

```diff
@@ -117,7 +117,7 @@
         'rest_framework.authentication.SessionAuthentication',
     ),
     'DEFAULT_FILTER_BACKENDS': (
-        'rest_framework.filters.DjangoObjectPermissionsFilter',
+        'resolwe.permissions.filters.ResolwePermissionsFilter',
```

Comment:
I was also considering this option, but came to the conclusion that we would have two pieces of code with exactly the same functionality (only difference is a join when filtering permissions - second commit in this PR).
So if we write our own function we should use it for all models. But why should we write it, if it is already written and we just need to modify it a bit?